src/moveit_visual_tools.cpp

```diff
@@ -1395,26 +1395,34 @@ bool MoveItVisualTools::publishRobotState(const std::vector<double> joint_positi
 }
 
 bool MoveItVisualTools::publishRobotState(const robot_state::RobotStatePtr& robot_state,
-                                          const rviz_visual_tools::colors& color)
+                                          const rviz_visual_tools::colors& color,
+                                          const std::vector<std::string>& highlight_links)
 {
-  return publishRobotState(*robot_state.get(), color);
+  return publishRobotState(*robot_state.get(), color, highlight_links);
 }
 
 bool MoveItVisualTools::publishRobotState(const robot_state::RobotState& robot_state,
-                                          const rviz_visual_tools::colors& color)
+                                          const rviz_visual_tools::colors& color,
+                                          const std::vector<std::string>& highlight_links)
 {
+  // when only a subset of links should be colored, the default message is used
```

Comment:
```suggestion
  // when only a subset of links should be colored, the default message is used rather than the cached solid robot messages
```

Reply:
also, i would be open to removing the cache feature entirely if that made sense.